hips/tiles/hipsdescription.py

```diff
@@ -0,0 +1,75 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HiPS description class -- provides methods for parsing the HiPS file properties."""
+
+__all__ = [
+    'HipsDescription',
+]
+
+from collections import OrderedDict
+
+
+class HipsDescription:
+    """This class provides methods for parsing the HiPS properties file.
+       It also provides multiple getters around the commonly used properties.
+    """
+
+    def __init__(self, properties: str) -> None:
+        super(HipsDescription, self).__init__()
+        self.properties = self.parse_file_properties(properties)
+
+    @classmethod
+    def parse_file_properties(cls, properties: str) -> OrderedDict:
+        """Returns an ordered dictionary containing the HiPS file properties.
+
+        This method parses the HiPS file properties and loads it in a dictionary,
+        then is casts it to `collections.OrderedDict` type.
+
+        Parameters
+        ----------
+        properties : str
+            HiPS file properties
+        
+        Returns
+        -------
+        dict_properties : OrderedDict
+        """
+        properties = properties.split('\n')
+        dict_properties = {}
+        for property in properties:
+            key_value = property.split('=')
+            try:
+                dict_properties[key_value[0].strip()] = key_value[1].strip()
+            except:
```

Comment:
Why is this `try-except` needed at all?
If it's not needed, please remove it.
If it's needed, catch the specific exception that occurs and leave a one-line comment why / when it occurs (e.g. by giving an example of a line where this happens.